docs/packaging-an-application/snapshots.md

```diff
@@ -25,9 +25,12 @@ Snapshots include customer console configuration, data from bind mounted volumes
 
 `script`: A bash script that will run on the server at the time of backup.
 
+`hidden`: A boolean flag that controls if the snapshots tile is shown on the Replicated dashboard.  Defaults to false.
```

Comment:
Let's be more explicit on "Defaults to false".  We intentionally made this show so that someone has to hide it. Let's call this out very clearly.